Test/SqlDom/Baselines170/JsonIndexTests170.sql

```diff
@@ -0,0 +1,38 @@
+CREATE JSON INDEX IX_JSON_Basic
+    ON dbo.Users (JsonData);
+
+CREATE JSON INDEX IX_JSON_SinglePath
+    ON dbo.Users (JsonData)
+    FOR ('$.name');
+
+CREATE JSON INDEX IX_JSON_MultiplePaths
+    ON dbo.Users (JsonData)
+    FOR ('$.name', '$.email', '$.age');
+
+CREATE JSON INDEX IX_JSON_WithOptions
+    ON dbo.Users (JsonData) WITH (FILLFACTOR = 90, ONLINE = OFF);
+
+CREATE JSON INDEX IX_JSON_Complete
+    ON dbo.Users (JsonData)
+    FOR ('$.profile.name', '$.profile.email') WITH (MAXDOP = 4, DATA_COMPRESSION = ROW);
+
+CREATE JSON INDEX IX_JSON_Filegroup
+    ON dbo.Users (JsonData)
+    ON [PRIMARY];
+
+CREATE JSON INDEX IX_JSON_Full
+    ON dbo.Users (JsonData)
+    FOR ('$.orders[*].amount', '$.orders[*].date') WITH (DROP_EXISTING = OFF, ALLOW_ROW_LOCKS = ON)
```

Comment:
Is DROP_EXISTING can be used with the JSON indexes? I guess it is valid for normal indexes!